content/blog/2017/2017-04-05-welcome-to-blue-ocean-editor.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean and the Pipeline Visual Editor"
+tags:
+- blueocean
+- ux
+- pipeline
+- tutorial
+- editor
```

Comment:
s/editor/screencast/

IMHO 'editor' isn't a terribly useful tag for post-metadata